_plugins/asset_path_tag/asset_path_tag.rb

```diff
@@ -0,0 +1,145 @@
+<<~aid*- coding: utf-8 -*- ruby -*- _plugins/asset_path_tag.rb *****************
+
+  A Jekyll plugin to output a relative URL for assets based on the post or page
+
+  MIT License copyright original author(s); minimally adapted for passing strict
```

Comment:
Note: Derivatives are expected to clearly state modifications; we need a place for this (realistically for future too, i find it crazy almost nobody does this as its part of agreement lol).